py/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.8.52
 
-- Add a data category for continuous profiling. ([#3284](https://github.com/getsentry/relay/pull/3284))
+- Add a data categories for continuous profiling. ([#3284](https://github.com/getsentry/relay/pull/3284),[#3303](https://github.com/getsentry/relay/pull/3303))
```

Comment:
That version is already released, new entry please.

Checking your last PR, you also updated the Relay changelog, maybe should also update that with this PR just so it's consistent.